CannyEdgeNode:
  ros__parameters:
    low_threshold: 50
    high_threshold: 150
